examples/update_bids_datasets.py

```diff
@@ -0,0 +1,140 @@
+"""
+========================
+11. Update BIDS datasets
+========================
+
+When working with electrophysiological data in the BIDS format, we usually
+do not have all the metadata stored in the ``Raw`` mne-python object.
+We can update the BIDS sidecare files via the ``update_sidecars`` function.
+
+In this tutorial, we show how ``update_sidecars`` can be used to update and
+modify BIDS-formatted data.
+
+"""
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
+
+###############################################################################
+# .. contents:: Contents
+#    :local:
+#    :depth: 1
+#
+# Imports
+# -------
+# We are importing everything we need for this example:
+import os.path as op
+from mne.datasets import somato
+import tempfile
+import json
+from mne_bids import (BIDSPath, read_raw_bids,
+                      print_dir_tree, make_report)
+from mne_bids.sidecar_updates import update_sidecars
```

Comment:
```suggestion
from mne_bids import update_sidecar
```

now this function should be quite straightforward:

```
for sc_fname in path.match():
     sc = json.load(sc_fname)
     sc.update(**values)
     json.dump(sc, sc_fname)
```

I don't get why such a small util function generates 400 lines of code.
If we are not careful of keeping things simple we are on a slippery slope.

Reply:
> I don't get why such a small util function generates 400 lines of code.

To be fair, 1/4 of this is tests, and then there's also an example, so the **effective** number of new lines of production code is much smaller than 400